mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -142,8 +142,11 @@ def test_io_set_raw(fnames, tmpdir):
     shutil.copyfile(op.join(base_dir, 'test_raw.fdt'),
                     negative_latency_fname.replace('.set', '.fdt'))
     event_id = {eeg.event[0].type: 1}
-    pytest.raises(ValueError, read_raw_eeglab, montage=montage, preload=True,
-                  event_id=event_id, input_fname=negative_latency_fname)
+    with warnings.catch_warnings(record=True) as w:
```

Comment:
BTW, now that we are pytesting, you can pytestify this code:
```python
with pytest.warns(UserWarning, match="have a sample index of -1."):
    # ...
```

Reply:
In fact this pattern will be required once #5409 is in, which is almost done